src/_shared/styled/index.ts

```diff
@@ -9,7 +9,6 @@ export { StyledDropzoneBox } from './StyledDropzoneBox';
 export { StyledHistoryCollapse } from './StyledHistoryCollapse';
 export { StyledLinkButton } from './StyledLinkButton';
 export { StyledListCard } from './StyledListCard';
-export { StyledListItemIcon } from './StyledListItemIcon';
```

Comment:
Binned this file since the styles within wouldn't apply anyway ¯\_(ツ)_/¯. I have noticed it with other CSS properties elsewhere - I think this is a MUI limitation.